requirements/edx/testing.in

```diff
@@ -30,7 +30,7 @@ factory_boy==2.8.1        # Library for creating test fixtures, used in many tes
 freezegun                 # Allows tests to mock the output of assorted datetime module functions
 httpretty                 # Library for mocking HTTP requests, used in many tests
 isort                     # For checking and fixing the order of imports
-moto==0.3.1               # Lets tests mock AWS access via the boto library
+moto==1.3.1               # Lets tests mock AWS access via the boto library
```

Comment:
Out of curiosity, why is this the new explicit version? Are there changes keeping us from the latest?

Reply:
`moto` pins other dependencies explicitly in it's setup.py and going to the latest caused conflicts with some of our other dependencies.  This version had the thing I needed(python 3 support) and did not cause an upgrade cascade.